components/Callout.tsx

```diff
@@ -27,15 +29,16 @@ const Callout = ({ title, icon, children }) => (
 
 const iconStyle = { color: 'inherit' };
 
-const getIcon = (icon) => {
-    switch (icon) {
-        case 'shield':
-            return <ShieldIcon style={iconStyle} />;
-        case 'link':
-            return <LinkTwoIcon style={iconStyle} />;
-        default:
-            return <InfoIcon style={iconStyle} />;
-    }
+interface DynamicIconProps {
+    iconName: string;
+}
+
+const DynamicIcon: React.FC<DynamicIconProps> = ({ iconName }) => {
+    const Icons = dynamic(() => import(`@100mslive/react-icons/dist/${iconName}.js`) as any, {
+        loading: () => <InfoIcon style={iconStyle} />
```

Comment:
Can we add this as a fallback? It's possible that some icon can get renamed/removed in the fututre